design-documents/graph-ql/coverage/catalog/compare-list.graphqls

```diff
@@ -0,0 +1,38 @@
+type ComparableItem {
+    productId: ID!
+    name: String!
+    priceRange: ProductPriceRange!
+    url: ProductUrl
+    images: [ProductImage]
+    values: [ProductAttribute]
+}
+
+type ComparableIndicator {
+    code: String!
+    title: String!
+}
+
+type CompareList {
+    items: [ComparableItem]
+    indicators: [ComparableIndicator]
+}
+
+type Customer{
+    compareList: CompareList
+}
+
+input CompareListInput {
+    id: ID!
+    items: [ID!]
+}
+
+type Mutation {
```

Comment:
Please combine these documents with existing proposal https://github.com/magento/architecture/blob/master/design-documents/graph-ql/coverage/product-compare.graphqls

Reply:
removed old file

Reply:
It may be confusing. We need to have only one document because they are competing.

Reply:
removed old document